src/apprt/gtk/headerbar_adw.zig

```diff
@@ -0,0 +1,74 @@
+const std = @import("std");
+const c = @import("c.zig").c;
+
+const Window = @import("Window.zig");
+const adwaita = @import("adwaita.zig");
+
+const HeaderBar = @import("headerbar.zig").HeaderBar;
+
+const AdwHeaderBar = if (adwaita.versionAtLeast(0, 0, 0)) c.AdwHeaderBar else anyopaque;
+const AdwWindowTitle = if (adwaita.versionAtLeast(0, 0, 0)) c.AdwWindowTitle else anyopaque;
+
+pub const HeaderBarAdw = struct {
```

Comment:
Wouldn't it be easier for downstream importers for this to be a file-level struct?

Reply:
Fixed.